sagemaker_neo_compilation_jobs/gluoncv_yolo/gluoncv_yolo_neo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I downloaded and unzipped the <code>VOCdevkit</code> directory and these train and val filepaths do not exist. Please update the paths or else the whole notebook will fail.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Are you referring to train.lst and val.lst? If so, then those get generated by prepare_dataset.py. They aren't part of the downloaded archive.

Reply:
I see. I looked at the CI logs and the cv2 module is missing. Please add `pip install opencv-python` since that is not a default module installed in SageMaker environments yet.

Reply:
Sure, I can add that. I'm curious though, I fired up a new notebook instance last night to test the existing version of the notebook and there were no issues in running it. Why would CI fail but not the instance?

Reply:
<p>There's a variety of kernels and environments with different modules pre-installed - I'd assume you were testing on one that has it installed, and CI uses one that doesn't.</p>